CharlieBackend.Api.UnitTes/CurrentUserTests.cs

```diff
@@ -42,56 +44,57 @@ public void ThrowErrorIfNotLoggined()
         [Fact]
         public void getAccountId()
         {
+            // Arrange
+            long expectedAccountId = 123;
+
             _mockIdentity.Setup(i => i.Claims)
                 .Returns(new Claim[]{
-                    new Claim("AccountId", "123") 
+                    new Claim("AccountId", expectedAccountId.ToString()) 
                 });
-            _httpContext.User = new ClaimsPrincipal(_mockIdentity.Object);
-            _httpContextAccessorMock.Setup(req => req.HttpContext).Returns(_httpContext);
 
-            _currentuserService.AccountId.Should().Be(123);
+            // Act & Assert
+            _currentuserService.AccountId.Should().Be(expectedAccountId);
         }
         [Fact]
         public void getEntityId()
```

Comment:
You're the first person I see to completely ignore the coding style of the rest of the project  and use _camelCase_ methods naming. 
Come on, it's not JavaScript. We use _PascalCase_ naming in C#.

Even better if you'd follow [the unit test naming convention](https://docs.microsoft.com/en-us/dotnet/core/testing/unit-testing-best-practices#naming-your-tests). You can also look at other unit tests for reference.

Reply:
Duplicate of https://github.com/ita-social-projects/WhatBackend/pull/504#discussion_r646455509